src/types.rs

```diff
@@ -10,8 +10,8 @@ use crate::sdk;
 
 pub type AccountId = String;
 pub type RawAddress = [u8; 20];
-pub type RawU256 = [u8; 32];
-pub type RawH256 = [u8; 32];
+pub type RawU256 = [u8; 32]; // Little-endian large integer type.
```

Comment:
Is it little endian? I thought EVM was always big endian

Reply:
Yes, it's pulled from here: https://docs.rs/ethereum-types/0.1.4/ethereum_types/

Reply:
Hmm, I'm finding conflicting info on this.

Reply:
My question was based on what I read in the Mastering Ethereum book

> All numbers in Ethereum are encoded as big-endian integers

https://github.com/ethereumbook/ethereumbook/blob/develop/06transactions.asciidoc

Reply:
In blake2f, they use little endian on parts too, not all of it. *shrug*

Reply:
@birchmd You were right with this. The documentation https://docs.rs/ethereum-types/0.1.4/ethereum_types/ confused me. Web assembly uses little-endian, but the EVM uses big-endian integers. So larger number were encoded wrongly in the `evm-bully` which also caused #122.